rtloader/test/datadog_agent/datadog_agent.go

```diff
@@ -229,3 +232,16 @@ func writePersistentCache(key, value *C.char) {
 func readPersistentCache(key *C.char) *C.char {
 	return (*C.char)(helpers.TrackedCString("somevalue"))
 }
+
+//export obfuscateSQL
+func obfuscateSQL(rawQuery *C.char, errResult **C.char) *C.char {
+	s := C.GoString(rawQuery)
+	obfuscatedQuery, err := obfuscate.NewObfuscator(nil).ObfuscateSQLString(s)
```

Comment:
We can just not call anything here and return/set the correct value depending on the input. The obfuscate code is already tested in it's own module. We're only testing how rtloader handles the returned value here.

Reply:
Ok, updated the remove the dependency